Firebase/Auth/Source/AuthProvider/Phone/FIRPhoneAuthProvider.m

```diff
@@ -271,7 +271,7 @@ - (void)verifyClientAndSendVerificationCodeToPhoneNumber:(NSString *)phoneNumber
       callback(nil, error);
       return;
     }
-    FIRSendVerificationCodeRequest *request;
+    FIRSendVerificationCodeRequest * _Nullable request;
```

Comment:
Is this necessary? I thought it's always default to NULL

Reply:
request is default to nil under ARC so there's no need to initialize it with nil (that is, request = nil by default).

But _Nullable is a compiler annotation to mark it as nullable.

These two are different schematics. The former is a runtime behavior. The latter is for analyze/checks during compilation stage.

Reply:
Actually, I meant it's default to nullable and set to NULL. I don't see an analyze error without adding `_Nullable`. While it's clearer for sure.